README.md

```diff
@@ -4,9 +4,12 @@ This images is intended to be used to build and deploy applications to a k8s clu
 within gitlab-ci.
 
 This image includes:
- * Docker client 17.12.0-ce
- * Docker-compose 1.22.0
- * Google cloud sdk 306.0.0
- * Helm 2.14.3 (helm binary)
- * Helm 3.3.0 (helm3 binary)
+ * Docker client 20.10.7-ce
+ * Docker-compose 1.29.2
+ * Google cloud sdk 346.0.0
+ * Helm 3.7.1 (helm3 binary)
```

Comment:
`helm3` is no more a static binary, but only a symlink to keep the compatibility

```suggestion
 * Helm 3.7.1
```